scripts/core/editor3/components/annotations/AnnotationInput.jsx

```diff
@@ -62,21 +63,33 @@ export class AnnotationInput extends Component {
     }
 
     componentDidMount() {
+        const annotationTypes = ng.get('metadata').values.annotation_types;
```

Comment:
Is there any reason why you are doing this here? I was thinking it would make more sense to do it in the constructor. It feels odd to set the state _again_ after the component mounts. Let's just start with the state already prepared before this happens.